platform/entanglement/src/services/LocatingObjectDecorator.js

```diff
@@ -37,7 +37,7 @@ define(
             this.$q = $q;
         }
 
-        LocatingObjectDecorator.prototype.getObjects = function (ids) {
+        LocatingObjectDecorator.prototype.getObjects = function (ids, options) {
```

Comment:
I really think we should avoid options objects on the whole, they tend to obfuscate code. Let's just pass the abort signal down by itself.